### ISSUE TITLE
`docker run` commands seem to use `-t` by mistake

### ISSUE THREAD

**Issue by rrthomas** [junaruga/ci-multi-arch-test#48]
Note that for `docker run`, `-t` means `--tty`, not `--tag`. This may not hurt, but it doesn't seem to be intentional!

Thanks for this very useful template project.

**Comment by junaruga** [comment]
Yeah. `docker run`'s `-t` might not be needed to be used in `.travis.yml`. I have not checked it yet.
